Assets/Scripts/Gameplay/Player/PlayerMovement.cs

```diff
@@ -63,6 +58,12 @@ void Update()
     private void FixedUpdate()
     {
         body.velocity = new Vector2 (horizontal * speed, vertical * speed);
```

Comment:
not your code from this review, but I would encourage you to consider saving the Vect2 once in the file instead of creating and deleting every update in here a new instance.